src/v2/Components/Onboarding/GenesStep/GeneSearchResults.tsx

```diff
@@ -43,7 +43,6 @@ const NoResultsContainer = styled.div`
 
 class GeneSearchResultsContent extends React.Component<Props, null> {
   private excludedGeneIds: Set<string>
-  followCount: number = 0
```

Comment:
I really like how you were able to remove these properties from the classes - every little bit of simplification helps! 🏆